liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/utils/MavenProjectUtil.java

```diff
@@ -16,8 +16,9 @@
 package io.openliberty.tools.maven.utils;
 
 import java.io.File;
+import java.util.HashMap;
 import java.util.Iterator;
-import java.util.List;
```

Comment:
This is causing a compile error e.g. in the travis build  https://travis-ci.com/github/OpenLiberty/ci.maven/jobs/324026004 :
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project liberty-maven-plugin: Compilation failure
[ERROR] /home/travis/build/OpenLiberty/ci.maven/liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/utils/MavenProjectUtil.java:[126,9] cannot find symbol
[ERROR]   symbol:   class List
[ERROR]   location: class io.openliberty.tools.maven.utils.MavenProjectUtil
```

Reply:
Hmm, not sure what happened there!